Airports/EGNV/Sector.txt

```diff
@@ -3,8 +3,11 @@ DISPLAY:NVAPP:NVAPP:NVTWR
 DISPLAY:NVTWR:NVAPP:NVTWR
 
 SECTORLINE:NVAPP
+DISPLAY:NVAPP:East:NVAPP
+DISPLAY:NTAPP:NTAPP:NVAPP
+DISPLAY:NVAPP:NVAPP:NTAPP
 DISPLAY:East:East:NVAPP
```

Comment:
I would suggest East is only interested in seeing the NV APP outline if it's not just because NT APP is covering top-down. I would create an Only NVAPP in the EGNV ownership file, and then change this to Only NVAPP